packages/js/src/indexables-page/components/empty-state.js

```diff
@@ -0,0 +1,43 @@
+
+import PropTypes from "prop-types";
+
+import IconSearch from "../../../images/icon-search.svg";
+import Indexation from "./indexation";
+
+/**
+ * Renders a page in case the SEO optimization process has not been performed.
+ *
+ * @returns {WPElement} A div containing the empty state page.
+ */
+function EmptyState( { setIndexingState } ) {
```

Comment:
I feel like EmptyState does not clearly convey what this component renders. Perhaps we should call it `NeedsIndexation` or `IndexationView` or something?